scripts/storybook/src/utils.js

```diff
@@ -217,7 +217,9 @@ function getPackageStoriesGlob(options) {
   const projects = getAllProjects();
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31226&commitId=65c2c5737d09b6e2482b551722bf4d404318bf65&env=prod&clientType=FLUENTUI)


  <details><summary><b>DatePicker - Disabled</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker - Disabled.Without Value.default.chromium.png|32| Changed|
|DatePicker - Disabled.Without Value.hover datepicker.chromium.png|32| Changed|
|DatePicker - Disabled.Without Value.click.chromium.png|32| Changed|

</details>


  <details><summary><b>DetailsHeader</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DetailsHeader.Root.default.chromium.png|244| Changed|
|DetailsHeader.Root.hoverDraggable.chromium.png|292| Changed|
|DetailsHeader.Root.hoverFrozenLast.chromium.png|288| Changed|
|DetailsHeader.Root.hoverFrozenFirst.chromium.png|293| Changed|
|DetailsHeader.Root.dropHint.chromium.png|288| Changed|
|DetailsHeader.Root.borderWhileDragging.chromium.png|292| Changed|

</details>


  <details><summary><b>DetailsList</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DetailsList.Compact.hover.chromium.png|97| Changed|
|DetailsList.Root.hover.chromium.png|97| Changed|
|DetailsList.Compact.click.chromium.png|86| Changed|
|DetailsList.Compact.default.chromium.png|51| Changed|
|DetailsList.Root.default.chromium.png|51| Changed|
|DetailsList.Root.click.chromium.png|86| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>